recipes/default.rb

```diff
@@ -4,10 +4,13 @@
 
 pkgs = %w[
   glibc.i686
-  glibc
+  glibc.x86_64
+  libgss.x86_64
   libgcc.i686
   compat-libstdc++-33
   compat-db
 ]
 
-package pkgs
+package pkgs do
+  action :upgrade
```

Comment:
`upgrade` is problematic with glibc - surely `:install` is good enough?

Reply:
nope - needed upgrade otherwise the package versions don't match between the i686 and x86_64 versions